applications/machine_learning/src/events/ml_result_event.c

```diff
@@ -17,10 +17,28 @@ static int log_ml_result_event(const struct event_header *eh, char *buf, size_t
 			event->label, event->value, event->anomaly);
 }
 
+static int log_ml_result_signin_event(const struct event_header *eh, char *buf, size_t buf_len)
+{
+	const struct ml_result_signin_event *event = cast_ml_result_signin_event(eh);
+
+	return snprintf(buf, buf_len, "module: \"%s\" %s ml_result_event",
+		(const char *)module_id_get(event->module_idx),
```

Comment:
`module_id_get` may return NULL for improper module_idx. Maybe it's worth to assert it here.

Reply:
ok